modules/csharp_api/csharp/CSharpAPI.csproj

```diff
@@ -0,0 +1,80 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!--生成多框架库;-->
+    <TargetFrameworks>net6.0;net48</TargetFrameworks>
+    <GenerateDocumentationFile>True</GenerateDocumentationFile>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>    
+    <PackageId>OpenVINO.CSharp.win</PackageId>
+    <!--设置NuGet包信息-->
+    <Title>OpenVINO C# API</Title>
+    <Version>3.0.122-test-8</Version>
+    <Authors>Guojin Yan</Authors>
+    <Company>Guojin Yan</Company>
+    <Product>OpenVINO C# API</Product>
+    <Description>基于C#平台调用OpenVINO套件部署深度学习模型。
```

Comment:
Please, use English in documentation, because it is very confusing for not Chinese speakers to read mix of Chinese and English in one sentence